source/algorithms.tex

```diff
@@ -5272,25 +5272,30 @@
 \pnum
 \remarks
 If two sequences have the same number of elements and their corresponding
-elements are equivalent, then neither sequence is lexicographically
+elements (if any) are equivalent, then neither sequence is lexicographically
 less than the other.
 If one sequence is a prefix of the other, then the shorter sequence is
 lexicographically less than the longer sequence.
 Otherwise, the lexicographical comparison of the sequences yields the same
 result as the comparison of the first corresponding pair of
 elements that are not equivalent.
 
+\pnum
+\begin{example}
+The following sample implementation satisfies these requirements:
 \begin{codeblock}
 for ( ; first1 != last1 && first2 != last2 ; ++first1, (void) ++first2) {
   if (*first1 < *first2) return true;
   if (*first2 < *first1) return false;
 }
 return first1 == last1 && first2 != last2;
 \end{codeblock}
+\end{example}
 
 \pnum
-\remarks\ An empty sequence is lexicographically less than any non-empty sequence, but
+\begin{note} An empty sequence is lexicographically less than any non-empty sequence, but
```

Comment:
When do we use the normal "note" environment and when do we use the `\Notes` element?